python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -119,12 +119,17 @@ class LogisticRegression(Base,
         If 'elasticnet' is selected, OWL-QN will be used if l1_ratio > 0,
         otherwise L-BFGS will be used.
     tol: float (default = 1e-4)
-       The training process will stop if current_loss > previous_loss - tol
+        Tolerance for stopping criteria.
+        The exact stopping conditions depend on the chosen solver.
```

Comment:
Aren't we using the same stopping condition for both `min_lbfgs` and `min_owlqn`?

Reply:
Yes, but I assume here some day we add other methods besides QN :)

Reply:
Sounds fair, thanks for clarifying.